store/tikv/gcworker/gc_worker_test.go

```diff
@@ -289,3 +292,19 @@ func (s *testGCWorkerSuite) TestCheckGCMode(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(useDistributedGC, Equals, true)
 }
+
+func (s *testGCWorkerSuite) TestRunGCJob(c *C) {
+	gcSafePointCacheInterval = 0
+	err := RunGCJob(context.Background(), s.store, 0, "mock", 1)
+	c.Assert(err, IsNil)
+	gcWorker, err := NewGCWorker(s.store, s.pdClient)
+	c.Assert(err, IsNil)
+	gcWorker.Start()
+	gcWorker.(*GCWorker).runGCJob(context.Background(), 0, 1)
```

Comment:
Do we need to check the `runGCJob` is running normally?

Reply:
The worker is mocked and not async running. I think it is not necessary.